ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh

```diff
@@ -16,6 +16,8 @@ ocm login --url "${OCM_LOGIN_ENV}" --token "${OCM_TOKEN}"
 CLUSTER_ID=$(cat "${SHARED_DIR}/cluster-id")
 echo "Deleting cluster: ${CLUSTER_ID}"
 
+sleep 86400
```

Comment:
For the time sleep, you want to keep the cluster for 1 day rignt?

Reply:
yes.  I just want to set up a OSD cluster,  this PR won't be merged. thanks a lot